hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -383,6 +383,7 @@
                 export PROJECT="k8s-jkns-e2e-gke-prod"
                 export CLOUDSDK_BUCKET="gs://cloud-sdk-testing/rc"
                 export JENKINS_USE_SERVER_VERSION="y"
+                export CLOUDSDK_API_ENDPOINT_OVERRIDES_CONTAINER="https://container.googleapis.com/"
```

Comment:
It's just these two. And I double-checked that the two staging tests were set correctly as well.